Engine/source/console/engineFunctions.h

```diff
@@ -108,7 +108,17 @@ struct _EngineFunctionDefaultArguments< void(ArgTs...) > : public EngineFunction
       std::tie(std::get<I + (sizeof...(ArgTs) - sizeof...(TailTs))>(args)...) = defaultArgs;
    }
    
+#ifdef _MSC_VER == 1910
```

Comment:
This got reintroduced that'll cause compiler warning spam on non-Windows builds.